composer.json

```diff
@@ -80,16 +80,21 @@
         "egulias/email-validator": "~1.2"
     },
     "autoload": {
-        "psr-0": { "Symfony\\": "src/" },
+        "psr-0": {
+            "Symfony\\Bridge\\Doctrine\\": "src/",
+            "Symfony\\Bridge\\Monolog\\": "src/",
+            "Symfony\\Bridge\\ProxyManager\\": "src/",
+            "Symfony\\Bridge\\Swiftmailer\\": "src/",
+            "Symfony\\Bridge\\Twig\\": "src/",
+            "Symfony\\Bundle\\": "src/",
+            "Symfony\\Component\\": "src/"
```

Comment:
We already recommend it: http://symfony.com/doc/current/book/performance.html#use-composer-s-class-map-functionality